lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -212,9 +212,9 @@
           - "Service memory reservation in format C(<number>[<unit>]). Number is a positive integer.
```

Comment:
I'm not really familiar with swarm, but should this also be changed?
```suggestion
          - "Service memory limit in format C(<number>[<unit>]). Number is a positive integer.
```

Reply:
Oh, probably!

Reply:
Yes it should!